scripts/reports/bundle-size/time-series.mjs

```diff
@@ -5,7 +5,7 @@ const branch = process.env.GIT_BRANCH;
 const fileName = 'bundle-size-time-series.csv';
 
 function isMasterBranch() {
-  return branch === 'origin/master';
+  return branch === 'origin/v2';
```

Comment:
I don't understand. What about the bundle-size for PRs after this ? Why change all of these 'master' to 'v2' ?

Reply:
Because essentially, starting now: the `v2` branch is master for `v2` development, so for every PR starting and targeting `v2`, the tools & co should consider 'v2' as the master branch to have proper comparaison